pallets/committee/src/types.rs

```diff
@@ -31,7 +28,6 @@ impl<T: Config> Proposal<T> {
 }
 
 #[derive(PartialEq, Eq, Clone, RuntimeDebug, Encode, Decode)]
-#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
 /// Defines what sub-type a member belongs to.
```

Comment:
we need these derives in order to put membersips into genesis configs

Reply:
`GeneisConfig` only requires `AccountId` now, we don't need to write `MemberType` in it anymore

Reply:
ah, you're right, that's even better.